CHANGELOG.md

```diff
@@ -9,6 +9,17 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 The following changes have been implemented but not released yet:
 
+### Breaking Changes
+
+- Support for Node.js v12.x has been dropped as that version has reached
+  end-of-life.
+- We've removed support for the registrationAccessToken / initialAccessToken
+  when performing dynamic client registration. This feature wasn't fully
+  implemented, and does not really make sense for Dynamic Client Registration in
+  the context of Solid — you're probably better
+  off using a [Public Client Identifier
+  Document](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/authenticate-client/)
```

Comment:
May I suggest we move this into bugfixes as it's really a cleanup for a not fully implemented functionality. I'm going to tweak the text a bit
```suggestion
- We've cleaned up dynamic client resgistration  by removing 
  support for the registrationAccessToken / initialAccessToken
  when performing the registration flow as this feature was never fully
  implemented. For Solid apps we recommend the use 
  of a [Public Client Identifier
  Document](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/authenticate-client/)
```